application/src/gui/modules/settings.py

```diff
@@ -0,0 +1,149 @@
+from PySide2.QtWidgets import QSlider  # type: ignore
+from PySide2.QtWidgets import QVBoxLayout  # type: ignore
+from PySide2.QtWidgets import QWidget  # type: ignore
+from PySide2.QtWidgets import QLabel  # type: ignore
+
+width = 300
+height = 220
+font = 10
+opaci_percent = 100
+opaci = opaci_percent/100
+frame = False
+top = False
+
+class FontSpin(QSpinBox):
+    def __init__(self):
+        QSpinBox.__init__(self)
+        self.setMinimum(5)
+        self.setMaximum(25)
+        self.setValue(font)
+        self.setSingleStep(1)
+
+
+class OpaciSpin(QSpinBox):
+    def __init__(self):
+        QSpinBox.__init__(self)
+        self.setMinimum(1)
+        self.setMaximum(100)
+        self.setValue(opaci_percent)
+        self.setSingleStep(10)
+        self.setSuffix("%")
+class FrameCheck(QCheckBox):
+    def __init__(self):
+        QCheckBox.__init__(self)
+        self.setChecked(frame)
+
+
+class TopCheck(QCheckBox):
+    def __init__(self):
+        QCheckBox.__init__(self)
+        self.setChecked(top)
+
+class ApplyButton(QPushButton):
+    def __init__(self):
+        QPushButton.__init__(self)
+        self.setText("Apply")
+
+
+        def Apply():
+            font = FontSpin.value
+            opaci_percent = OpaciSpin.value
+            frame = FrameCheck.checkState
+            top = TopCheck.checkState
+            width = TopBar.width_text
+            height = TopBar.height_text
+        self.clicked.connect(Apply)
+
+
+class TopBar(QWidget):
+    def __init__(self):
+        QWidget.__init__(self)
+        self.layout = QHBoxLayout()
+        self.layout.setSpacing(0)
+        self.layout.setMargin(0)
+
+        width_name = QLabel()
+        width_name.setText("Window width:    ")
+        self.layout.addWidget(width_name)
+
+        WidthSlider = QSlider(Qt.Horizontal)
+        WidthSlider.setMinimum(300)
+        WidthSlider.setMaximum(1920)
+        WidthSlider.setValue(width)
+        WidthSlider.setSingleStep(10)
+        self.layout.addWidget(WidthSlider)
+
+        WidthSlider.valueChanged[int].connect(self.valuechange)
+        width_text = QLabel()
+        width_text.setNum(width)        
+        self.layout.addWidget(width_text)
+
+        height_name = QLabel()
+        height_name.setText("\n Window height:    ")
+        self.layout.addWidget(height_name)
+
+        HeightSlider = QSlider(Qt.Horizontal)
+        HeightSlider.setMinimum(220)
+        HeightSlider.setMaximum(1080)
+        HeightSlider.setValue(height)
+        HeightSlider.setSingleStep(10)
+        self.layout.addWidget(HeightSlider)
+
+        HeightSlider.valueChanged[int].connect(self.valuechange)
+        height_text = QLabel()
+        height_text.setNum(height)        
+        self.layout.addWidget(height_text)
+
+        self.setLayout(self.layout)
+
+    def valuechange(self, value):
+        self.__init__(value)   # FCK INIT CALLED TWICE, i was looking for solution but can't find anything ;_;
```

Comment:
I'm not sure what you want achieve there ^.^?

Reply:
I need to update the value on the screen every time the user move slider that is when the value changes.